daemons/controld/controld_cib.c

```diff
@@ -593,9 +595,11 @@ append_restart_list(lrmd_event_data_t *op, struct ra_metadata_s *metadata,
     }
 
     digest = calculate_operation_digest(restart, version);
-    /* Add "op-force-restart" and "op-restart-digest" to indicate the resource supports reload,
-     * no matter if it actually supports any parameters with unique="1"). */
-    crm_xml_add(update, XML_LRM_ATTR_OP_RESTART,
+    /* Add PCMK__XA_OP_FORCE_RESTART and "op-restart-digest" to indicate the
+     * resource supports reload, no matter if it actually supports any
+     * parameters with unique="1").
```

Comment:
while we're here, change that last line to `reloadable parameters`

Reply:
assuming just `reloadable parameters` and not `reloadable parameters with unique="1"`

Reply:
done